2016-11-17-mvcc-in-tikv.md

```diff
@@ -10,23 +10,23 @@ categories: ['Engineering']
 
 ## Introduction to concurrency control
 
-Transaction isolation is important for database management system. Because database should provide an illusion that the user is the only one who connects to the database, which greatly simplifies application development. But, the concurrency controlling problems like data races must be resolved since there will be a lot of connections to the database. Due to this background, the database management system (DBMS) ensures that the resulting concurrent access patterns are safe, ideally by serializablity.
```

Comment:
Replace this sentence with the concept of transaction to be more logic and relevant in the context.